subsys/net/l2/ethernet/ethernet_stats.c

```diff
@@ -29,7 +29,7 @@ static int eth_stats_get(u32_t mgmt_request, struct net_if *iface,
 
 		eth = net_if_get_device(iface)->driver_api;
 		len_chk = sizeof(struct net_stats_eth);
-		src = eth->stats;
+		src = eth->get_stats(iface);
```

Comment:
Missing a check whether the driver implements `get_stats()` here, otherwise it'll crash.
```
+         if (eth->get_stats == NULL) {
+                 return -ENOENT;
+         }
```

Reply:
Thanks for catching that.